src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/EligibilityStatusCode.java

```diff
@@ -2,7 +2,6 @@
 
 public enum EligibilityStatusCode {
     INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS,
-    INVALID_COMPANY_APPOINTMENTS_MORE_THAN_FIVE_OFFICERS,
```

Comment:
This constant is still present in the web layer but I'll create another PR to tidy this up later. The fact that API no longer returns this code does not break anything.